### PR TITLE
Add sample apy calculations to v2

### LINKED ISSUE #1225: Add sample APY passive calculations to v2 docs
- [x] Remove DF "belt system" references
- [x] Basic restructure. Onboard users w/ veOCEAN & passive rewards first, THEN active rewards.
- [x] Get 4yr + 6mo sample calculations setup into V2 Docs
- [x] Describe how APY is calculated

### PR DESCRIPTION
**Pull request by idiom-bytes**
Fixes #1225 

Changes proposed in this PR:
- [x] Clean up belts
- [x] Clean up some DF basics for the launch
- [x] Addresses APY estimate calcs

### CODE REVIEW

**Review by alexcos20 — DISMISSED** [review]
the google sheet provides wrong estimations

When calculating rewards for 2 years/1 year/6 month, used veOcean balance is wrong(starts from line 2, as 4 year lock would do) -> leads to incorrect APY (for 6 months the doc says 15.26% when in reality it's just 0.9%)

Fixes:
- for 6 months, sum rewards from week 184 to week 207
- for 1 year, sum rewards from week ....
- for 2 years, sum rewards from week ....

File: rewards/veocean.md [hunk 1]
```diff
@@ -1,10 +1,9 @@
 ---
 description: >-
-  Let's discuss the "ve" in veOCEAN for our last jutsu before earning a black
-  belt in Data Farming knowledge!
+  Learn the basic moves to start kicking a** Data Farming
```
Comment (CasianaOcean) [inline rewards/veocean.md:3]: Learn the basic moves to start kicking a** at Data Farming

File: user-guides/how-to-df-estimate-apy.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+### Estimating Passive APY
+
+To make it easier to estimate your APY, [we have created a simple spreadsheet](https://docs.google.com/spreadsheets/d/1zzuW5pBbX6j6hZL_XtJDtSR2-rDHa_LGOEwgoQ4D8lk/edit?usp=sharing) that let's you easily estimate your Passive APY.
+
+We have provided 2 sheets as an example of locking-up 10,000 OCEAN for:
+1. A 4-year investment period
+2. A 6-mo investment period
+
+These examples are meant to be as simple as possible so anyone can understand them. They are naive investment strategies and are meant to provide the reader some examples to build upon. Please copy this spreadsheet and modify as you wish.
```
Comment (CasianaOcean) [inline user-guides/how-to-df-estimate-apy.md:22]: The above are simplified examples meant for everyone to understand. They are naive investment strategies and are meant to provide you, the reader, with some examples to build upon. You can use these as a reference to create your own plan, so feel free to make a copy of this spreadsheet and customize it to your needs and wants.

**Review by CasianaOcean — COMMENTED** [review]
Seems like a good approach, I am unclear wrt the belts system, is it still a thing or has it been replaced? Left my notes in the review.

File: rewards/df-basic.md [hunk 3]
```diff
@@ -1,14 +1,14 @@
 ---
-description: Learn the basic moves to start kicking a** Data Farming
+description: Show you can complete all User-Guides as the last steps to master Data Farming!
```
[the remark this answers is not in the document]
Comment (idiom-bytes) [reply rewards/df-basic.md:2]: Great, that's what's implemented.

**Comment by idiom-bytes** [comment]
Thanks for the comments @CasianaOcean, I have fixed them.

**Comment by mihaisc** [comment]
What was the issue with the belts?

**Comment by idiom-bytes** [comment]
```
What was the issue with the belts?
```
@mihaisc, a belt-system (like karate) was added but:
1. It didn't quite explain the system
1. The files weren't named/updated to reflect this too much
1. It became a pain in the ass to update/reorder things

I simply removed all references to this and improved the ordering/onboarding.

**Pull request merged**